testing/scripts/test_rolling_updates.py

```diff
@@ -226,21 +226,25 @@ def test_rolling_update10(self, namespace, api_gateway):
         run(f"kubectl delete -f ../resources/graph6svc.json -n {namespace}", shell=True)
 
 
-@pytest.mark.flaky(max_runs=3)
+@pytest.mark.flaky(max_runs=2)
 @with_api_gateways
 @pytest.mark.parametrize(
-    "from_deployment,to_deployment",
+    "from_deployment,to_deployment,change",
     [
-        ("graph1.json", "graph2.json"),  # New image version
-        ("graph1.json", "graph3.json"),  # New image version and new name of container
-        ("graph1.json", "graph4.json"),  # New resource request but same image
-        ("graph1.json", "graph5.json"),  # Update with multi-deployment new model
-        ("graph1.json", "graph8.json"),  # From v1alpha2 to v1
-        ("graph7.json", "graph8.json"),  # From v1alpha3 to v1
+        ("graph1.json", "graph2.json", True),  # New image version
+        (
+            "graph1.json",
+            "graph3.json",
+            True,
+        ),  # New image version and new name of container
+        ("graph1.json", "graph4.json", True),  # New resource request but same image
+        ("graph1.json", "graph5.json", True),  # Update with multi-deployment new model
+        ("graph1.json", "graph8.json", True),  # From v1alpha2 to v1
+        ("graph7.json", "graph8.json", False),  # From v1alpha3 to v1
```

Comment:
Does `change=False` mean that the current pod doesn't get deleted? How is the update from `v1alpha3` to `v1` happening?

Reply:
As I see it as there is no change between these resources it gets changed automatically by the k8s api to the stored version so there is literally no difference here.

Reply:
Got it! That makes sense.